hudi-common/src/main/java/org/apache/hudi/common/table/view/AbstractTableFileSystemView.java

```diff
@@ -104,9 +104,9 @@ private String getPartitionPathFor(HoodieBaseFile baseFile) {
   /**
    * Initialize the view.
    */
-  protected void init(HoodieTableMetaClient metaClient, HoodieTimeline visibleActiveTimeline) {
+  protected void init(HoodieTableMetaClient metaClient, HoodieTimeline visibleActiveCompletedTimeline) {
```

Comment:
I will pull this down and ensure all callers to this method are passing in completed timeline.